fang_examples/simple_async_worker/src/main.rs

```diff
@@ -12,16 +12,18 @@ async fn main() {
 
     log::info!("Starting...");
 
-    let mut queue = AsyncQueue::connect("postgres://postgres:postgres@localhost/fang", NoTls, true)
-        .await
-        .unwrap();
+    let mut queue = AsyncQueue::connect(
+        "postgres://postgres:postgres@localhost/fang",
+        NoTls,
```

Comment:
instead of passing arguments into `connect`. I suggest the following
1. Init `AsyncQueue` with the builder
2. Add an additional field to `AsyncQueue` - `connected: boolean`
3. connect func won't accept any arguments

Reply:
okay :D

Reply:
Solve in [75dc2c4](https://github.com/ayrat555/fang/pull/54/commits/75dc2c4abd5d99376c50e91ce68f423b1cf91d85)